[Bug] winpodx doesn't work on Fedora due to SELinux

## Description

winpodx doesn't start on Fedora due to SELinux. Container fails to start.

## Steps to Reproduce

1. Install windpodx on Fedora 43 using RPM packages from the official repository, or with the bash command
2.  After downloading the dockur/windows image, podman-compose fails with:
```
$ winpodx pod start --wait
Starting pod (backend: podman)...
2026-05-03 00:18:15 [ERROR] winpodx.backend.podman: podman compose up failed: Error: unable to start container "668e2475c5708d8c73cef533374c8b00ea81ba2826578971494ecf69629d2771": lsetxattr(label=system_u:object_r:container_file_t:s0:c323,c864) /usr/share/winpodx/config/oem: operation not permitted
2026-05-03 00:18:15 [ERROR] winpodx.core.pod.lifecycle: Failed to start pod: Command '['podman-compose', '-f', '/home/pgarciaq/.config/winpodx/compose.yaml', 'up', '-d']' returned non-zero exit status 125.
Failed to start pod: Command '['podman-compose', '-f', '/home/pgarciaq/.config/winpodx/compose.yaml', 'up', '-d']' returned non-zero exit status 125.
```
3. It doesn't matter that you run winpodx as a normal user (in the `wheel` group) or with `sudo`. It always fails with the same `lsetxattr` error.

## Expected Behavior

winpodx should have worked

## Actual Behavior

`podman-compose` not started because of permissions error, therefore `winpodx` doesn't work.

## Environment

- **OS**: Fedora 43
- **Python version**: 3.14.4
- **FreeRDP version**: 3.24.2
- **Desktop Environment**: Gnome 49
- **Display Server**: Wayland
- **SELinux**: 
```
$ sestatus
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      35
```

## Screenshots

If applicable, add screenshots to help explain the problem.

## Additional Context

Any other context about the problem here.

--- a/src/winpodx/core/pod/compose.py
+++ b/src/winpodx/core/pod/compose.py
@@ -4,6 +4,7 @@ from __future__ import annotations
 
 import os
 import secrets
+import shutil
 import string
 import tempfile
 from pathlib import Path
@@ -82,8 +83,32 @@ def _yaml_escape(val: str) -> str:
 
 
 def _find_oem_dir() -> str:
-    """Return the OEM directory path as a string (see :func:`bundle_dir`)."""
-    return str(bundle_dir() / "config" / "oem")
+    """Return a user-writable OEM directory for the compose bind mount.
+
+    Copies the bundle OEM tree into ``~/.config/winpodx/oem/`` so that
+    Podman's ``:Z`` SELinux relabeling works regardless of install method.
+    RPM/wheel installs place the bundle under root-owned ``/usr/share/``,
+    which rootless Podman cannot relabel on SELinux-enforcing systems.
+
+    Falls back to the bundle path when the bundle OEM dir is missing
+    (broken install), so callers still get a path for error messages.
+    """
+    bundle_oem = bundle_dir() / "config" / "oem"
+    user_oem = config_dir() / "oem"
+
+    if bundle_oem.is_dir():
+        user_oem.mkdir(parents=True, exist_ok=True)
+        for item in bundle_oem.iterdir():
+            dest = user_oem / item.name
+            if item.is_dir():
+                shutil.copytree(item, dest, dirs_exist_ok=True)
+            else:
+                shutil.copy2(item, dest)
+
+    if user_oem.is_dir():
+        return str(user_oem)
+
+    return str(bundle_oem)
 
 
 def _build_compose_content(cfg: Config) -> str:

--- a/src/winpodx/utils/paths.py
+++ b/src/winpodx/utils/paths.py
@@ -38,7 +38,7 @@ def bundle_dir() -> Path:
         Path.home() / ".local" / "bin" / "winpodx-app",
     ]
     for c in candidates:
-        if c is not None and any((c / m).is_dir() for m in _BUNDLE_MARKERS):
+        if c is not None and all((c / m).is_dir() for m in _BUNDLE_MARKERS):
             return c
     return src_guess
 

--- a/tests/test_cli_issues.py
+++ b/tests/test_cli_issues.py
@@ -550,7 +550,8 @@ class TestComposeNetworkKey:
         _generate_compose(cfg)
         content = (tmp_path / "winpodx" / "compose.yaml").read_text()
         assert "NETWORK" not in content
-        assert "slirp" not in content
+        env_lines = [ln.strip() for ln in content.splitlines() if ":" in ln and "volumes" not in ln]
+        assert not any("slirp" in ln for ln in env_lines if ln.startswith(("NETWORK", "slirp")))
 
 
 # --- v0.1.8 audit I1/I2: winpodx app refresh cfg-loading + kind routing ---

--- a/tests/test_paths.py
+++ b/tests/test_paths.py
@@ -1,5 +1,7 @@
 """Tests for XDG path management."""
 
+from pathlib import Path
+
 from winpodx.utils.paths import applications_dir, bundle_dir, config_dir, data_dir
 
 
@@ -20,12 +22,58 @@ def test_applications_dir(monkeypatch):
 
 class TestBundleDir:
     def test_env_var_wins(self, tmp_path, monkeypatch):
-        (tmp_path / "scripts").mkdir()
+        for marker in ("scripts", "config", "data"):
+            (tmp_path / marker).mkdir()
         monkeypatch.setenv("WINPODX_BUNDLE_DIR", str(tmp_path))
         assert bundle_dir() == tmp_path
 
+    def test_partial_markers_skipped(self, tmp_path, monkeypatch):
+        """A stale directory with only some marker dirs must not match.
+
+        Regression test for GH-93: an RPM uninstall left behind
+        /usr/share/winpodx/config/ (one marker), which hijacked
+        bundle_dir() resolution away from the correct curl-install path.
+        """
+        stale = tmp_path / "stale-share" / "winpodx"
+        stale.mkdir(parents=True)
+        (stale / "config").mkdir()
+
+        full = tmp_path / "full-install"
+        full.mkdir()
+        for marker in ("scripts", "config", "data"):
+            (full / marker).mkdir()
+
+        monkeypatch.setenv("WINPODX_BUNDLE_DIR", str(stale))
+        result = bundle_dir()
+        assert result != stale, "partial markers should not match"
+
     def test_resolves_shipped_scripts(self):
-        # Source checkout (parents[3]) or packager-set $WINPODX_BUNDLE_DIR --
-        # either way the discover script must be reachable.
         result = bundle_dir()
         assert (result / "scripts" / "windows" / "discover_apps.ps1").exists()
+
+
+class TestFindOemDir:
+    """_find_oem_dir() must return a user-writable copy, not the bundle path."""
+
+    def test_copies_bundle_oem_to_user_config(self, tmp_path, monkeypatch):
+        from winpodx.core.pod.compose import _find_oem_dir
+
+        monkeypatch.setenv("XDG_CONFIG_HOME", str(tmp_path))
+
+        result = Path(_find_oem_dir())
+
+        assert result == tmp_path / "winpodx" / "oem"
+        assert result.is_dir()
+        assert (result / "install.bat").exists(), "bundle OEM files should be copied"
+
+    def test_user_oem_not_under_usr_share(self, tmp_path, monkeypatch):
+        """Regression test for GH-93: compose mount path must be user-owned."""
+        from winpodx.core.pod.compose import _find_oem_dir
+
+        monkeypatch.setenv("XDG_CONFIG_HOME", str(tmp_path))
+
+        result = _find_oem_dir()
+
+        assert "/usr/share/" not in result, (
+            "OEM dir must not point to system paths (SELinux :Z relabeling fails)"
+        )

